__sample__/prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="../__sample__/prep-html-links/home-page.html">
```

Comment:
Unfortunately this link does not work. Mostly because you do not have a `home-page.html` file.